terraform/environments/digital-prison-reporting/modules/glue_job/main.tf

```diff
@@ -180,7 +180,7 @@ resource "aws_iam_role_policy_attachment" "glue_policies" {
 }
 
 resource "aws_cloudwatch_log_group" "log_group" {
-  name              = "/aws-glue/jobs/${var.name}-sec-config"
```

Comment:
Having checked test environment, there are some log groups like for example /aws-glue/jobs/dpr-reporting-hub-test-sec-config and dpr-domain-refresh-test-sec-config which are set to expire in a week.
How are those different than other log groups with the sec-config suffix that do not expire?

Reply:
@gavriil-g-moj  default is to set to expire "log_group_retention_in_days=7 days"

Reply:
@gavriil-g-moj the ones  with the suffix `sec-config` (i.e. the ones without an expiry period) have a typo where the job name occurs twice.
For example, the dpr-reporting-hub-test should use:

`/aws-glue/jobs/dpr-reporting-hub-test-sec-config`

instead of

`/aws-glue/jobs/dpr-reporting-hub-test-sec-config-dpr-reporting-hub-test-sec-config`

Reply:
If there is a typo in the name and the job name occurs twice then as per my understanding by removing the -sec-config will not fix the duplication issue.